geoportal/c2cgeoportal_geoportal/views/mapserverproxy.py

```diff
@@ -174,7 +174,7 @@ def _proxy_callback(
     ) -> Response:
         if self.request.matched_route.name.endswith("_path"):
             url = url.clone()
-            url.path = self.request.path
+            url.path = "/".join(self.request.matchdict["path"])
```

Comment:
For me, this is suspect, is the `QGIS_SERVER_LANDING_PAGE_PREFIX` configured correctly? 
To `mapserv_proxy/qgis`

Reply:
Yes, but we still should request http://qgisserver:8080/index.html not http://qgisserver:8080/mapserv_proxy/qgis/index.html
Maybe I miss something, but it works like that.

Reply:
I setted the value of QGIS_SERVER_LANDING_PAGE_PREFIX in demo.
But it did not changed anything, I do not really understand if we should use QGIS_SERVER_LANDING_PAGE_PREFIX env var or X-Qgis-Service-Url.

I see in doc that:
QGIS_SERVER_LANDING_PAGE_PREFIX is Landing page base URL path prefix
It is used here: https://github.com/qgis/QGIS/blob/7ae1b318731fda9a98f29bb8d795a274d7ab0124/src/server/services/landingpage/qgslandingpagehandlers.cpp#L77

But I see this about X-Qgis-Service-Url:
https://github.com/qgis/QGIS/blob/9563b040ef37aee51db54a8074227ca4e2886767/src/server/qgsserverprojectutils.h#L441:L461

In all case API still return http://qgisserver:8080

Reply:
In fact it does not work anymore when settings QGIS_SERVER_LANDING_PAGE_PREFIX

Reply:
I do some test with QGIS server alone (with our image) and As I understand we should have `QGIS_SERVER_LANDING_PAGE_PREFIX` and here we should have the pull path...